treeherder/perf/alerts.py

```diff
@@ -120,3 +121,86 @@ def generate_new_alerts_in_series(signature):
                         'new_value': new_value,
                         't_value': t_value
                     })
+
+
+def os_relevance(alert):
+    platforms = [
+        {
+            'name': 'windows10',
+            'relevance': 5
+        },
+        {
+            'name': 'windows7',
+            'relevance': 4
+        },
+        {
+            'name': 'linux',
+            'relevance': 3
+        },
+        {
+            'name': 'osx',
+            'relevance': 2
+        },
+        {
+            'name': 'android',
+            'relevance': 1
+        }
+    ]
+
+    for platform in platforms:
+        if platform['name'] in alert.series_signature.platform.platform:
+            return platform['relevance']
+
+
+def select_from_each_platform(sorted_all_alerts, cherry_pick_config):
```

Comment:
We should provide test coverage for this new feature.
I suggest you write the tests in the tests/perfalert/test_create_alerts.py module.

Reply:
Ok, the unit tests for the new functions will be added in test_create_alerts.py module.